### PR TITLE
Add missing certificate fields to inspect output

### PR DESCRIPTION
**Pull request by tommy-56**
Closes Issue #446 in CLI (https://github.com/smallstep/cli/issues/446). Email name constraint missing. This displays the missing email name constraint mentioned in the issue.

### CODE REVIEW

**Comment by dopey** [comment]
https://github.com/smallstep/cli/issues/446#issuecomment-876824658

**Comment by tommy-56** [comment]
Example Output of step certificate inspect with changes to certInfo.go
Certificate:
    Additional Cert info Here ...
            X509v3 Name Constraints:
                Permitted:
                  DNS: Pexample.com
                  Email: jane@doe.com
                  URI: https://doe.com
                  IP Range: 1.2.3.0/24
                Excluded:
                  DNS: ExcludexampleDNS.net
                  Email: janeexcluded@doe.org
                  URI: https://EXdoe.org
                  IP Range: 3.2.1.0/24
    Signature Algorithm: ECDSA-SHA256
         30:46:02:21:00:f2:c9:87:6d:97:dc:25:06:1e:30:38:ab:ab:
         10:73:09:26:16:50:2f:57:65:b3:14:94:79:f3:04:d3:8e:68:
         c3:02:21:00:c9:59:6c:89:77:a2:46:69:45:7a:28:1a:c0:37:
         8a:aa:e1:78:d2:74:bc:78:50:a1:a6:89:84:19:5b:96:a1:b4
		 
------------
Multiple Constraints
Certificate:
    Additional Cert info Here... 
            X509v3 Name Constraints:
                Permitted:
                  DNS: Pexample.com, PermitMe.org, permitMe2.net
                  Email: jane@doe.com, permit@smallstep.com
                  URI: https://doe.com, https://jonPermitURI.com
                  IP Range: 1.2.3.0/24, 5.6.7.0/24
                Excluded:
                  DNS: ExcludexampleDNS.net, ExcludeMe.org
                  Email: janeExcluded@doe.org, johnExclude@smallstep.com
                  URI: https://EXdoe.org, EXJon.net
                  IP Range: 3.2.1.0/24, 1.2.3.0/24
    Signature Algorithm: ECDSA-SHA256
         30:45:02:20:50:ac:60:bb:d9:46:13:ed:23:1f:fd:d4:1d:ee:
         8f:90:ca:53:0a:50:8e:91:20:2c:3e:f4:f3:ca:78:2a:40:8d:
         02:21:00:e2:ce:34:29:20:5a:85:c1:40:e1:10:3f:0e:e1:d7:
         97:fb:40:24:43:32:0c:c4:cd:4d:ae:72:94:3f:f3:b5:fd

File: certinfo.go [hunk 1]
```diff
@@ -492,12 +492,63 @@ func CertificateText(cert *x509.Certificate) (string, error) {
 						buf.WriteString("\n")
 					}
 					if len(cert.PermittedDNSDomains) > 0 {
-						buf.WriteString(fmt.Sprintf("%16sPermitted:\n%18s%s", "", "", cert.PermittedDNSDomains[0]))
+						buf.WriteString(fmt.Sprintf("%16sPermitted:\n%18sDNS: %s", "", "", cert.PermittedDNSDomains[0]))
```
Comment (dopey) [inline certinfo.go:495]: Seems like `Permitted` will only be printed if dnsdomains is not empty. Doesn't take into account that dnsDomains might be empty while other stuff is not. Same is true for `Excluded` below.

**Review by dopey — CHANGES_REQUESTED** [review]
see comment in code.

**Review by dopey — APPROVED** [review]
lgtm

**Pull request merged**